vocabularies/UI.xml

```diff
@@ -1389,6 +1390,52 @@ The trend is
         </Member>
       </EnumType>
 
+      <Term Name="Note" Type="UI.NoteType" Nullable="false" AppliesTo="EntityType">
+        <Annotation Term="Core.Description" String="The annotated entity type can be visualized in a reuse component for notes" />
+      </Term>
+      <ComplexType Name="NoteType">
+        <Property Name="Title" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="Title of the note" />
+        </Property>
+        <Property Name="ContentType" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.IsMediaType" />
+          <Annotation Term="Core.Description" String="MIME type of the note" />
+        </Property>
+        <Property Name="Content" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="Content of the note, as a string" />
+        </Property>
+        <Property Name="Type" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="Type of the note" />
+        </Property>
+        <Property Name="TypeDescription" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="Description for the type of the note" />
+        </Property>
+        <Property Name="Language" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="Language of the note" />
+        </Property>
+        <Property Name="LanguageName" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="Name of the language of the note" />
+        </Property>
+        <Property Name="ObjectNodeType" Type="Edm.String" Nullable="false">
```

Comment:
Similar to the `Created*` and `LastChanged*` fields, the `ObjectNodeType` refers to the `sapObjectNodeType`. This is not something specific to the Notes component.
In CDS we already have annotations like `@ObjectModel.sapObjectNodeType[Reference]`, maybe we should define something common here too.